src/grid/interfaces.tsx

```diff
@@ -65,5 +65,11 @@ export namespace GridProps {
      * The number of grid elements to push the column to the right by.
      */
     push?: number | GridProps.BreakpointMapping;
+
+    /**
+     * Test ID of the grid column.
+     * Assigns this value to the `data-testid` attribute of the column's root element.
```

Comment:
Should we instead say like: "so that in can be found with findColumnByTestId(testId) test util"? The current one rather describes the internal implementation as we don't expect someone to rely on the attribute in with some custom queries.

Reply:
We use a template for this prop JSDoc and the problem with mentioning a test util finder here is that: there are interfaces like `Option` where it is being used by different components and they don't necessarily implement the selectors exactly the same way.

> describes the internal implementation as we don't expect someone to rely on the attribute

I think we should expect the customers to do that. Why `data-testid` should be an internal implementation detail? The customers can either query directly or use test utils finders.

To which element `data-testid` will be assigned is an internal detail, but the attribute it self should not be so.